subsys/bluetooth/services/cgms/Kconfig.cgms

```
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig BT_CGMS
+	bool "Enable GATT continuous glucose service"
+	select BT_NRF_SERVICES
+
+if BT_CGMS
+
+config BT_CGMS_NUMBER_MEAS_RECORD_MAX
```

I am now thinking that the number word is redundant here and we shouldn't have MAX as a suffix (e.g. BT_MAX_CONN)

```suggestion
config BT_CGMS_MAX_MEAS_RECORD
```

Sorry, for not getting it right in my first review comment